<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="main_window_tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="employees_tab">
     <attribute name="title">
      <string>Employees</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="employee_list_label">
         <property name="text">
          <string>Employees</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="employees_list"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>441</width>
        <height>511</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="employee_data_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>441</width>
         <height>501</height>
        </rect>
       </property>
       <widget class="QCalendarWidget" name="employee_calendar">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>330</y>
          <width>301</width>
          <height>171</height>
         </rect>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="remove_employee_button">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>0</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove Employee</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_employee_button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Employee</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>42</y>
          <width>261</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="employee_name_label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="employee_surname_label">
             <property name="text">
              <string>Surname:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="employee_salary_label">
             <property name="text">
              <string>Salary $/h:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="employe_id_label">
             <property name="text">
              <string>Employee_id:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="employee_name_edit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="employee_surname_edit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="employee_salary_edit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="employee_id_edit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tickets_tab">
     <attribute name="title">
      <string>Tickets</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucar">
    <property name="title">
     <string>Save</string>
    </property>
   </widget>
   <addaction name="menucar"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
